spec/features/apo_spec.rb

```diff
@@ -74,13 +77,17 @@
     click_button 'Add'
 
     fill_in 'Title', with: 'New APO Title'
+    select 'Stanford', from: 'Default Object Rights'
     fill_in 'Default Copyright statement', with: 'New copyright statement'
     fill_in 'Default Use and Reproduction statement', with: 'New use statement'
     select 'Attribution No Derivatives 3.0 Unported', from: 'Default use license'
     click_button 'Update APO'
+    expect(page).to have_text 'Actions' # wait for form to render
 
     click_on 'Edit APO'
+    expect(page).to have_text 'Add group' # wait for form to render
     expect(find_field('Title').value).to eq('New APO Title')
+    expect(find_field('Default Object Rights').value).to eq('stanford')
```

Comment:
and now this worked on jcoyne's laptop, but not mine.  I gave up and hope circleci works.